projects/demo/src/modules/customization/routable/eager/examples/2/dialog.component.ts

```diff
@@ -1,8 +1,9 @@
 import {ChangeDetectionStrategy, Component} from '@angular/core';
 
 @Component({
-    selector: 'tui-example-dialog-content',
+    standalone: true,
+    selector: 'tui-dialog',
```

Comment:
Could it be the same as Taiga UI dialogs' selector ? 🤔

Reply:
fixed